### PR TITLE
[DA-168] Use property instead of hardcoded url in web.xml cors configuration

### PR DESCRIPTION
**Pull request by jomrazek**
default value is /* you can specify own value with -Durl-pattern=value

### CODE REVIEW

**Comment by jbartece** [comment]
:+1:

**Comment by jomrazek** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Comment by jbartece** [comment]
retest this please

File: pom.xml [hunk 1]
```diff
@@ -64,6 +64,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <jboss.bom.version>6.3.2.GA</jboss.bom.version>
+        <url-pattern>/*</url-pattern>
```
Comment (jbartece) [inline pom.xml]: Could you put here more descriptive name, please? I'd suggest to use corsUrlPattern

**Comment by janinko** [comment]
:+1:

**Pull request merged**